apps/preview/next/pages/showcases/OrderSummary/OrderSummary.tsx

```diff
@@ -0,0 +1,93 @@
+
+  return (
+    <div className="md:shadow-lg md:rounded-md md:border md:border-neutral-100">
+      <div className="flex justify-between items-end bg-neutral-100 md:bg-transparent py-2 px-4 md:px-6 md:pt-6 md:pb-4">
+        <p className="typography-headline-4 font-bold">Order summary</p>
+        <p className="typography-text-base font-semibold">(Items: {orderDetails.items})</p>
+      </div>
+      <div className="px-4 pb-4 mt-3 md:px-6 md:pb-6 md:mt-0">
+        <div className="flex justify-between typography-text-base pb-4">
+          <div className="flex flex-col grow pr-2">
+            <p>Items Subtotal</p>
+            <p className="ml-auto typography-text-xs text-neutral-500">Original Price</p>
+            <p className="ml-auto typography-text-xs text-secondary-700">Savings</p>
+            <p className="my-2">Delivery</p>
+            <p>Estimated Sales Tax</p>
+          </div>
+          <div className="flex flex-col text-right">
+            <p>{formatPrice(itemsSubtotal())}</p>
+            <p className="typography-text-xs text-neutral-500">{formatPrice(orderDetails.originalPrice)}</p>
+            <p className="typography-text-xs text-secondary-700">{formatPrice(orderDetails.savings)}</p>
+            <p className="my-2">{formatPrice(orderDetails.delivery)}</p>
+            <p>{formatPrice(orderDetails.tax)}</p>
+          </div>
+        </div>
+        {!!promoCode && (
+          <div className="flex items-center py-4 border-t border-neutral-200">
+            <p>PromoCode</p>
+            <SfButton size="sm" variant="tertiary" className="ml-auto mr-2" onClick={() => removePromoCode()}>
+              Remove
+            </SfButton>
+            <p>{formatPrice(promoCode)}</p>
+          </div>
+        )}
+        <div className="flex gap-x-2 py-4 border-y border-neutral-200 mb-4">
+          <SfInput
+            value={inputValue}
+            placeholder="Enter promo code"
+            wrapperClassName="grow"
+            onChange={(event) => setInputValue(event.target.value)}
+          />
+          <SfButton variant="secondary" onClick={() => checkPromoCode()}>
+            Apply
+          </SfButton>
+        </div>
+        <p className="px-3 py-1.5 bg-secondary-100 text-secondary-700 typography-text-sm rounded-md text-center mb-4">
+          You are saving ${Math.abs(orderDetails.savings).toFixed(2)} on your order today!
+        </p>
+        <div className="flex justify-between typography-headline-4 font-bold pb-4 mb-4 border-b border-neutral-200">
+          <p>Total</p>
+          <p>{formatPrice(totalPrice())}</p>
+        </div>
+        <SfButton className="w-full"> Place order and pay </SfButton>
+        <div className="typography-text-sm mt-4 text-center">
+          <span>By placing my order, you agree to our </span>
+          <SfLink href="/"> Terms and Conditions </SfLink>
+          <span>and our</span>
+          <SfLink href="/"> Privacy Policy. </SfLink>
```

Comment:
```suggestion
          By placing my order, you agree to our <SfLink href="/">Terms and Conditions</SfLink> and our{' '}
          <SfLink href="/">Privacy Policy.</SfLink>
```
Blank spaces are in wrong place

Reply:
https://github.com/vuestorefront/storefront-ui/pull/2579/commits/0857b9464bd93c263ac286d9636be4fc976cf218